### DISCUSSION TITLE
Provide language override feature for plugins

### DISCUSSION THREAD

**Discussion by robertgdev** [filamentphp/filament#3929] [Ideas]
Let's assume that I've installed a plugin but for whatever reason, I want to change the translation files. Right now, this doesn't seem possible without editing the package source under /vendor. 

It would be great if it were possible to copy the package translation files under the main /resources/lang directory and then change any translations which I want to update. Ideally, this would not have to be the full translation file but just a subset of the translatable strings so that Filament could: 

1. Load the standard plugin translation file(s)
2. Then load the custom files under /resources/lang
3. merge item 1 (above) into/over item 2 (above) to arrive at a final/custom list of translation items for the plugin.

This would also have the following benefits: 

1. Your customizations could be committed to git without having to commit/alter the /vendor directory
2. Plugins would still be upgradable as your custom translations would be maintained outside of the /vendor directory

I realize that this feature depends on the spatie package tools, but maybe something like would still be possible.

**Comment by danharrin** [comment]
Hey, maybe I'm wrong but isnt this possible already? Publish through `vendor:publish`, and Laravel will automatically merge in the translations. This is a feature within Filament core packages too
